pac4j-couch/pom.xml

```diff
@@ -26,6 +26,11 @@
 			<artifactId>org.ektorp</artifactId>
 			<version>${ektorp.version}</version>
 		</dependency>
+        <dependency>
```

Comment:
Indeed, the `pom.xml` files are just a mess in terms of indentation. No problem to fix that.

That said, I'd prefer to have a 4-spaces indentation for XML files in the editorconfig as this is what was used so far (when it was used).